redis/api.w

```diff
@@ -0,0 +1,25 @@
+pub interface IRedis extends std.IResource {
+  inflight url(): str;
+  inflight set(key:str, value: str): void;
+  inflight get(key: str): str?;
+  inflight hset(key: str, field:str, value: str): void;
+  inflight hget(key: str, field: str): str?;
+  inflight sadd(key: str, value: str): void;
+  inflight smembers(key: str): Array<str>?;
+  inflight del(key: str): void;
+}
+
+// TODO: remove this once we have a proper way to extend interfaces
+pub interface IRedisClient extends IRedis {
+  inflight disconnect(): void;
+  // remove them
```

Comment:
??

Reply:
This bug was fixed today https://github.com/winglang/wing/pull/5074

Reply:
@subh-cs can you confirm this has been resolved?

Reply:
No, it is not resolved. Currently getting an different error message. checkout dev channel, I've posted it there.

Reply:
Hi @eladb, it is a bug https://github.com/winglang/wing/issues/5089, though I know it is similar to this https://github.com/winglang/wing/issues/5052.